apps/demo/src/app/pages/notification/examples/02-create-notification/index.ts

```diff
@@ -0,0 +1,33 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {NotificationService} from '@ng-web-apis/notification';
+import {filterPermission, isDenied, PermissionsService} from '@ng-web-apis/permissions';
+import {map, switchMap} from 'rxjs/operators';
+
+@Component({
+    selector: 'notification-page-example-2',
+    templateUrl: './index.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NotificationPageExample2 {
+    readonly denied$ = this.permissions.state('notifications').pipe(map(isDenied));
+
+    constructor(
+        private readonly notifications: NotificationService,
+        private readonly permissions: PermissionsService,
+    ) {}
+
+    sendNotification(): void {
+        this.notifications
+            .requestPermission()
+            .pipe(
+                filterPermission('granted'),
```

Comment:
`filter(isGranted)` would also work, right? I wonder if we need that `filterPermission` operator 🤔

Reply:
I have the same doubts too.
Let's delete it :)